metadata-ingestion/src/datahub/ingestion/source/powerbi.py

```diff
@@ -141,6 +142,9 @@ class PowerBiAPIConfig(EnvBasedSourceConfigBase):
     extract_reports: bool = pydantic.Field(
         default=True, description="Whether reports should be ingested"
     )
+    extract_orphan_datasets: bool = pydantic.Field(
+        default=False, description="Whether orphan datasets should be ingested"
```

Comment:
Minor: It might be better to describe in which cases datasets will be considered "orphaned" in the docs. I can totally see someone asking "what does orphaned mean?" while reading this doc

Reply:
You're totally correct. I'll add comment to the code and recipe at least.